scripts/apps/authoring/views/article-edit.html

```diff
@@ -167,6 +181,23 @@
     <div class="text-editor" sd-html-preview="item.body_html" ng-if="!_editable"></div>
 </div>
 
+<div class="field body" ng-if="!isMediaType && schema.body_html && editor.body_html.editor3" order="{{editor.body_html.order}}" sd-validation-error="error.body_html" data-required="schema.body_html.required" sd-width="{{editor.body_html.sdWidth || 'full'}}">
+  <label translate>Body</label>
+    <span sd-character-count data-item="item.body_html" data-html="true"></span>
+    <span sd-word-count data-item="item.body_html" data-html="true"></span>
+    <div id="bodyhtml"
+         sd-editor3
+         data-config="{disableEditing: isPublished(item), multiBlockEdition: true, tabindex: editor.body_html.order}"
```

Comment:
Does `multiBlockEdition` still makes sense with editor3 ? Maybe this state depends more on the available editor features